pkg/interface/src/views/components/ProfileOverlay.tsx

```diff
@@ -14,7 +14,7 @@ import {
   Icon
 } from '@tlon/indigo-react';
 import RichText from './RichText';
-import { withLocalState } from '~/logic/state/local';
+import useLocalState from '~/logic/state/local';
```

Comment:
shouldn't we be using settings state here? this isn't in localstorage anymore

Reply:
<img width="515" alt="image" src="https://user-images.githubusercontent.com/3999320/110722250-d0b59180-81c6-11eb-8481-0be5f7932daa.png">

uh should we

Reply:
`state.dark` is changed locally using an eventlistener, so it makes sense in Author. same with toggleOmnibox. but for ViewProfile and this ProfileOverlay it seems worth changing to use `settings.calm` selector stuff.

Reply:
Ok I changed this component. Talking with @liam-fitzgerald about the other stuff